osmosis-1/osmosis.zone_assets.json

```diff
@@ -3641,6 +3641,70 @@
       "osmosis_verified": false,
       "listing_date_time_utc": "2024-04-14T01:25:00Z",
       "_comment": "ReStake DAO $RSTK"
+    },
+    {
+      "chain_name": "composable",
+      "base_denom": "ibc/F9D075D4079FC56A9C49B601E54A45292C319D8B0E8CC0F8439041130AA7166C",
+      "path": "transfer/channel-1279/transfer/channel-52/wei",
+      "osmosis_unlisted": true,
+      "osmosis_verified": false,
+      "_comment": "Ethereum $ETH"
```

Comment:
Without a transferMethods section on these assets, the ibc transfers will by default just go to Composable, and not all the way to Ethereum. Is this the desired behaviour for Withdraw? (and vice versa for Deposit--can only deposit from Composable, but not able to deposit from Ethereum.)

My understanding was that Osmosis would be added as a destination network on Mantis to enable transfers across the full path between Ethereum and Osmosis.

Reply:
ty

added this section, lmk if I am missing anything else

Reply:
Not a strict requirement, but it would be nice if the URLs accepted parameters that preselected the asset.

Reply:
Can I make this in a future PR if possible? Once the assets have been listed